### ISSUE TITLE
Incorrect image produced when transparent meshes are used in the presence of other transparent geometry.

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#2121]
The basic problem is that the image does not look right when a Mesh plot is made transparent and there are other transparent plots in the picture. When the viewer renders such a scene in hardware, the transparent plots' alpha looks to be modified by the opacity of the lines, making opaque geometry under the lines show more brightly. When running in parallel on a multi-domain dataset, the problems are worse. In that case, various parts of the mesh lines are replaced with the background color. This probably has to do with how the domains are composited.

Transparency relies on the transparency actor for depth sorting. In parallel, the actor is also used to render transparent geometry in a second pass. I'm guessing that since the Mesh plot has its own renderer, and does not seem to make use of the transparency actor, that VisIt is not able to correctly render the transparent geometry in all of these cases. Results might be better if all the mesh lines were added into the transparency actor.

This looks *really* bad.

It might also be related to #1487.

See the attached script and result images for a reproducer.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Brad Whitlock
Original creation: 05/29/2014 06:32 pm
Original update: 07/30/2014 12:48 pm
Ticket number: 1860
